### ISSUE TITLE
Add Chrome sync feature

### ISSUE THREAD

**Issue by itskdog** [CatBlock/catblock#154]
Idea: sync settings (and optionally custom filters) via Chrome Sync storage. This can be done by altering the functions used to set and get settings from localStorage.

Size limitations need to be kept in mind, as do transferring existing settings. Perhaps a run-once on first update, and user setting to enable, in case they already have multiple installs, to avoid overwriting.

**Comment by catherdersoftware** [comment]
Yes. I currently use the Google Drive trick (as suggested on AdBlock) for a custom whitelist but it's cumbersome.
